pages/zustand/index.tsx

```diff
@@ -1,200 +1,3 @@
-import * as THREE from 'three'
```

Comment:
Why was the main page for zustand removed?

Reply:
We're keeping the demos separate from the docs. Maybe we could add an extra "Website" button right next to the existing docs and repo links for the few libraries that do have separate pages like this? 🤔

Reply:
Sorry, i forgot to remove this comment after you answered me on discord

It makes perfect sense :)